Travis fix

ConchobarMacNessa opened the pull request:
4 of the tests were failing previously
Mostly, this was caused by the fact that the database made all keys lower case, so while our tests were looking for avgRating, we were actually passing in avgrating
relates #62

alice-carr committed:
fixed second test
we had refactored the functino to come in a slightly different order. the test now reflects this
relates #62

alice-carr committed:
fixed the next test
our tagNodeByTag function had changed slightly - it was taking in a different structure
I changed the test to reflect this
relates #62

alice-carr committed:
fixed memoryNodesAndLinks test
we had changed the function to procude more links - that is, the links between the source nodes
the test now reflects this
relates #62

alice-carr committed:
fixed getMemoryNodePositions test
we were passing in avgRating when we should have been passing in avgrating
relates #62

des-des commented on tests/src/client/node_transformations.js:
```diff
@@ -12,7 +11,7 @@ const {
 } = require('../../../src/client/node_transformations.js');
 
-test('binByTag', (t) => {
+test('binByTag', (t) => { // WORKING
```
remove `// WORKING`

des-des commented on tests/src/client/node_transformations.js:
```diff
@@ -33,7 +32,7 @@ test('binByTag', (t) => {
 });
 
-test('sortWithMax', (t) => {
+test('sortWithMax', (t) => { // WORKING
```
Here also

des-des commented on tests/src/client/node_transformations.js:
```diff
@@ -57,79 +56,118 @@ test('sortWithMax', (t) => {
-          avgRating: 8.2,
-          id: 67456535,
-          likes: 18,
+test('tagNodesByTag', (t) => { // WORKING
```
here

des-des commented on src/client/node_transformations.js:
```diff
@@ -14,13 +14,15 @@ const sortWithMax = (nodeArray) => {
 };
 
-const tagNodesByTag = (tagsArray, startingCx, startingCy, generateId) => {
+const tagNodesByTag = (tagsArray, startingCx, startingCy) => {
```
Okay, this is now called, `centralMaxNodesByTag` or something

alice-carr committed:
removed unnecc comments
relates #62

alice-carr committed:
renamed tagNodesByTag function
to centralNodesByTag, to better reflect what the function was doing
relates #62
